docs/director/fetch-requests.md

```diff
@@ -0,0 +1,20 @@
+# Fetch Requests
```

Comment:
We only support API Definitions for now, right? It should be also stated here.

And also we need to mention that currently the calls are synchronous, and in future we want to make them asynchronous.